internal/gqlparser/README.md

```diff
@@ -1,5 +1,18 @@
-gqlparser [![CircleCI](https://badgen.net/circleci/github/vektah/gqlparser/master)](https://circleci.com/gh/vektah/gqlparser) [![Go Report Card](https://goreportcard.com/badge/github.com/vektah/gqlparser/v2)](https://goreportcard.com/report/github.com/vektah/gqlparser/v2) [![Coverage Status](https://badgen.net/coveralls/c/github/vektah/gqlparser)](https://coveralls.io/github/vektah/gqlparser?branch=master)
-===
+# gqlparser (Details of library residing in OPA's internal)
```

Comment:
Can we track the commit somewhere? I mean, this month, I'll remember it's probably 2.5.0, but when winter comes, I might have forgotten all about it 😅

Reply:
Yep, I'll put that information in shortly.